armi/reactor/composites.py

```diff
@@ -3142,6 +3151,19 @@ def _genChildByLocationLookupTable(self):
         for child in self:
             self.childrenByLocator[child.spatialLocator] = child
 
+    def getBoundingCircleOuterDiameter(self, Tc=None, cold=False):
```

Comment:
Can we get a unit test for this? 

Thanks!

Reply:
done.